doc/io.rst

```diff
@@ -661,6 +661,34 @@ exporting your objects to pandas and using its broad range of `IO tools`_.
 
 .. _IO tools: http://pandas.pydata.org/pandas-docs/stable/io.html
 
+.. _io.PseudoNetCDF:
+
+
+Formats supported by PseudoNetCDF
```

Comment:
can you move this after PyNIO and before pandas? I think that would make a little more logical sense.

Reply:
Sure.